src/include/simeng/pipeline/FetchUnit.hh

```diff
@@ -20,7 +20,8 @@ enum class LoopBufferState {
 // Struct to hold information about a fetched instruction
 struct loopBufferEntry {
   // Encoding of the instruction
-  const uint64_t encoding;
+  // TODO could be updated to uint8_t encoding[4]
```

Comment:
Currently the only usage that I can find is passing to predecode which has the parameter type uint8_t*. This would prevent the slightly nasty reinterpret cast FetchUnit.cc:45

Reply:
Had a little play and it could be a bit nasty so will leave as is